src/main/java/org/opensearch/security/user/UserService.java

```diff
@@ -276,4 +274,51 @@ public static void saveAndUpdateConfigs(final String indexName, final Client cli
             throw ExceptionsHelper.convertToOpenSearchException(e);
         }
     }
+
+    public static List<String> listServiceAccounts() {
+
+        final SecurityDynamicConfiguration<?> internalUsersConfiguration = load(getConfigName(), false);
+
+        List<String> serviceAccounts = new ArrayList<>();
+        for (Map.Entry<String, ?> entry : internalUsersConfiguration.getCEntries().entrySet()) {
+
+            final InternalUserV7 internalUserEntry = (InternalUserV7) entry.getValue();
+            final Map accountAttributes = internalUserEntry.getAttributes();
+            final String accountName = entry.getKey();
+            if (accountAttributes.containsKey("service") && accountAttributes.get("service") == "true") {
+                serviceAccounts.add(accountName);
+            }
```

Comment:
You can rewrite this as:
```suggestion
        if (accountAttributes.getOrDefault("service", "false").equals("true")) {
                serviceAccounts.add(accountName);
            }
```
for better performance and readability.